src/include/coredumpctl.h

```diff
@@ -1,3 +1,6 @@
+#ifndef __CEPH_COREDUMPCTL_H
```

Comment:
please `git grep` for `pragma once`. also please don't define a macro starts with `__` . identifiers starting with `__` are [reserved](http://en.cppreference.com/w/cpp/language/identifiers) and their use is _technically_ undefined behavior.